CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+CHANGES IN VERSION 1.9.0 (from 1.8.1)
+===================================
+
+Appium 1.9.0 is a feature release, comprising multiple updates.
+
+#### General
+* Full W3C Specification support.
+* Add full beta of [Espresso driver](https://github.com/appium/appium-espresso-driver)
+  for Android automation (used by specifying `automationName` capability to be
+  `"Espresso"`).
+* Add driver for [Samsung Tizen devices](https://github.com/Samsung/appium-tizen-driver)
+* Add `-image` find element strategy,
```

Comment:
... which allows to match elements by their screenshoted examples. This feature requires OpenCV libraries and adapters to be installed on the machine where Appium server is running

Reply:
That should be documented elsewhere. I would give a link to such documentation in here, but I couldn't find any.

Reply:
I still need to write it, sorry!

Reply:
we also did find element strategy by android viewtag

Reply:
I see viewtag in the espresso driver. I don't find any reference to it in the Android or UiAutomator2 drivers.